### DISCUSSION TITLE
Javascript jupyter native support like python.

### DISCUSSION THREAD

**Discussion by liudonghua123** [microsoft/vscode-jupyter#9317] [Questions and Answers] [answered]
It would be nice if vscode support writing js along with markdown in `.js` file via the jupyter support natively.

I have write an issue https://github.com/microsoft/vscode-nodebook/issues/15. 

However, I can only write js code with markdown in a `.py` file right now. I hava installed `ijavascript` and `jp-babel` jupyter kernel for js support.

<img width="711" alt="image" src="https://user-images.githubusercontent.com/2276718/157604801-1bd581a1-a491-422e-9e56-323a34789a9b.png">

**Comment by DonJayamanne** [comment, marked as answer]
Thanks for the suggestion, we do have an issue to track a similar feature request here https://github.com/microsoft/vscode-jupyter/issues/273

You might be interested in the following extension, I'm the author of that, and have been considering adding support for something you have requested in there https://marketplace.visualstudio.com/items?itemName=donjayamanne.typescript-notebook. You don't need the ijavascript or jp-babel installed for that.

**Reply by DonJayamanne** [reply]
I'm going to mark this as answered as this feature request is tracked here https://github.com/microsoft/vscode-jupyter/issues/273
